### ISSUE TITLE
Cannot hide USWDS Notifications when bundling app via webpack-dev-server

### ISSUE THREAD

**Issue by dpgalloway** [uswds/uswds#4090]
### Describe the bug
When I have uswds installed via npm and attempt to use webpack to bundle my app and watch for changes, the USWDS notifications display several times upon building and after any file update, significantly slowing down build time.

### Steps to reproduce the bug
1. On an app using webpack-dev-server, npm install --save uswds
2. Run webpack-dev-server
3. USWDS notifications constantly appear in the terminal output during initial build and file changes

### Expected behavior
Preferably, no USWDS notifications appear in the terminal output - it's not a helpful place to view release notes as it is constantly updating and moving down the terminal.

### Screenshots
![image](https://user-images.githubusercontent.com/13645440/111221846-a84fdd80-85a0-11eb-95d5-760c1420d8f2.png)

### System setup
- **USWDS version:** 2.10.1
- **Device:** PC Laptop
- **Operating system:**  Windows 10
- **Browser and version:** N/A

### Additional context
I am using vue-cli with USWDS (which relies on webpack-dev-server), however I was also able to recreate the issue using plain webpack-dev-server. I tried updating the stats option in the webpack config (https://webpack.js.org/configuration/stats/), but not matter what I set it to, the USWDS notifications still appear.

**Comment by mejiaj** [comment]
@dpgalloway there's a setting to turn them off in [`_uswds-theme-general:40`](https://github.com/uswds/uswds-site/blob/develop/css/settings/_uswds-theme-general.scss#L40)

**Comment by dpgalloway** [comment]
@mejiaj This setting does not do anything when running with webpack.

**Comment by dpgalloway** [comment]
My mistake - I was pulling in a library that also had uswds as a dependency and needed to set `$theme-show-notifications: false` there as well.

**Comment by dpgalloway** [comment]
@mejiaj I may not be doing things exactly how uswds suggests and may need to refine my setup, but here's the info:

In my app, I have an app.scss file. At the top, I set a few theme variables, e.g. `$theme-show-notifications: false`. After setting variables, I have `@import '~uswds/dist/scss/uswds'`. I then import this file in my Vue main.js file.

I also have a _variables.scss file with `@import '~uswds/dist/scss/packages/required'`, and some app-specific css variables. I added a css loaderOptions config in vue.config.js to automatically pull in the variables scss to every Vue component. So inside Vue style tags, I can use any of the uswds helper functions, e.g. `background-color: color('secondary')`.

I don't use much that requires JS, but so far I pull JS in by component and not globally. I have a combobox component which has `import comboBox from 'uswds/src/js/components/combo-box'` in the Vue script tag, then in the Vue mounted hook:

```
  mounted() {
    // Initialize component with USWDS javascript
    comboBox.on();
  }
```

The library that uses USWDS is a vue component library that I also maintain. I'm currently working on 3 apps that use USWDS, but over the past year or two I've had to extend USWDS for example to have a Modal component (I see there's one now) and a more robust Table component. I created this library to share between the 3 apps. I don't fully import uswds in this library - I only import the `required` scss module. To fix this notifications issue, I just added `$theme-show-notifications: false` above the import statement for the required module.

Let me know if that helps. I'm happy to provide more details if you'd like. The apps I'm working on aren't open sourced yet but will be.
